chia/full_node/generator.py

```diff
@@ -62,9 +62,9 @@ def setup_generator_args(self: BlockGenerator) -> Tuple[SerializedProgram, Progr
     return self.program, args
 
 
-def run_generator(self: BlockGenerator, max_cost: int) -> Tuple[int, SerializedProgram]:
+def run_generator_mempool(self: BlockGenerator, max_cost: int) -> Tuple[int, SerializedProgram]:
     program, args = setup_generator_args(self)
-    return GENERATOR_MOD.run_safe_with_cost(max_cost, program, args)
+    return GENERATOR_MOD.run_mempool_with_cost(max_cost, program, args)
```

Comment:
I don't care that much but since "mempool" is not an adjective like "safe", these names I think are confusing.  Despite the added length, I think these should be `run_generator_mempool_mode` and `run_mempool_mode_with_cost` or maybe `run_with_cost_and_mempool_mode` for that second one.  It's much uglier though so if you disagree, I don't care to push it any further.

Reply:
I don't have a strong opinion. I also think that it's a bit redundant to split these modes out a two separate functions. It's a parameter at the lower level, so just having a named parameter `mempool_mode` that's `True` or `False` might be even more intuitive.